app/models/inventory_item.rb

```diff
@@ -1,2 +1,9 @@
 class InventoryItem < ActiveRecord::Base
+  using(:grossman)
+  after_initialize :init
```

Comment:
I should be flogged for using them.

I would use some kind of service object, like an interactor or hand rolled. Bear in mind that the UUID would actually be stored in a local gman_services table and not in grossman itself.